tests/emitters/gradient_aborescence_emitter_test.py

```diff
@@ -44,3 +44,25 @@ def test_dtypes(dtype):
                                          sigma0=1.0,
                                          step_size=1.0)
     assert emitter.x0.dtype == dtype
+
+
+def test_adhere_to_solution_bounds():
```

Comment:
These values don't seem to make sense - the GridArchive solution_dim is 10, but then the emitter takes in an x0 with 2 dimensions. Then tell_dqd takes in values that don't match with the measures. Since this test is not using the archive at all, I think you can simplify it to just calling the emitter. Also, you can simplify it to a one-dimensional solution so that it's easier to understand the bounds.